_guide/_pages/interviews/technical.md

```diff
@@ -61,89 +61,29 @@ to produce and defend a desired solution. Preface by saying there’s no right a
 
 Help guide the candidate to the right line of thinking, don’t allow them to get too bogged down in the wrong line of thinking — help them get back on track.
 
-* ["How would you design software to run elevators? Can you talk through a high-level design? How would accommodating different building types change your design?"](https://docs.google.com/document/d/1oYmx_93-mq2QrqICCo8SNk8hHmnPPonPA1kg0vhy540/edit#heading=h.tgd47iei6k4w)
+* ["Let's say you and your team are tasked to build a feature or an app that would support photo sharing. How would you approach the design and build for this?"](https://docs.google.com/document/d/1Z9A0WJFJj_Vo975RkuzQQazEOSJTPixiiahrtlNOBiI/edit#heading=h.tgd47iei6k4w)
```

Comment:
Nitpicking here — maybe "Imagine you and your team…" instead of "Let's say you and your team…".

Reply:
Also flagging this one for another look in the document.

Reply:
I love the "how would you approach the design and build for this" framing as opposed to "high-level design".